src/Factory/ZddPropertyExtractor.php

```diff
@@ -37,8 +37,6 @@ public function extractPropertiesFromClass(string $className): PropertyList
             }
 
             if ($propertyType->allowsNull()) {
-                $propertyList->addNullableProperty($propertyName);
```

Comment:
If I understand correctly, we should remove this condition completly no ?

Reply:
Yes we can remove it :-)

Reply:
After discussion, we need to keep it and add property with null value to be sure the property will not be set to not null